django_zxcvbn_password_validator/zxcvbn_password_validator.py

```diff
@@ -63,7 +63,13 @@ def append_translated_feedback(old_feedbacks, feedback_type, new_feedbacks):
         if user:
             for value in user.__dict__.values():
                 user_inputs.append(value)
-        results = self.zxcvbn_implementation(password, user_inputs=user_inputs)
+        try:
+            results = self.zxcvbn_implementation(password, user_inputs=user_inputs)
+        except ValueError as error:
+            # zxcvbn raises ValueError only if the password is too long.
+            raise ValidationError(
+                _("Your password exceeds the maximal length of 72 characters.")
```

Comment:
Shouldn't the translation file be updated ? Or is this already translated ?

Reply:
Yes, I just didn't want to change in two pull requests as that will cause conflicts.